packages/wrangler/src/__tests__/logout.test.ts

```diff
@@ -32,7 +32,7 @@ describe("logout", () => {
 			rest.post("*/oauth2/revoke", (_, response, context) => {
 				// Make sure that we made the request to logout.
 				counter += 1;
-				response.once(context.status(200), context.text(""));
+				return response.once(context.status(200), context.text(""));
```

Comment:
@JacobMGEvans would you be able to chime in on this from an MSW perspective?

Reply:
That return got overlooked and is needed 😮‍💨 , This however tells us 2 things the counter in the test doesnt show us what we need to know about the request AND our code isn't confirming that the logout was successful from a response standpoint, we likely just check that the files are removed.